pylint/checkers/strings.py

```diff
@@ -104,7 +105,11 @@
               "Used when a PEP 3101 format string uses a lookup specifier "
               "({a[1]}), but the argument passed for formatting "
               "doesn't contain or doesn't have that key as an attribute.",
-              {'minversion': (2, 7)})
+              {'minversion': (2, 7)}),
+    'W1308': ("Mixing string literals concatenation with comma-separated %s",
```

Comment:
I don't like a notion of mentioning "comma-separated list" / "comma-separated tuples". Are there other lists or tuples in Python?

My attempt at rephrasing would be to do something along lines of:
> "Implicit string concatenation found in {type_name} literal

Same goes to message name.

Reply:
Sounds better, thanks !